framework/src/task_manager/task_manager_core.c

```diff
@@ -1495,6 +1500,20 @@ static int taskmgr_init_task_manager(void)
 	return OK;
 }
 
+void task_manager_populate_builtin_list(const builtin_info_t builtin_list[], int builtin_list_count)
+{
+	tm_builtin_list = TM_ALLOC(builtin_list_count * sizeof(tm_builtin_info_t));
+	for (int i = 0; i < builtin_list_count; i++) {
```

Comment:
declare i in the start of the function separately

Reply:
Done

Reply:
Done.